04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -11,13 +11,14 @@ category: spatio-temporal-workshop
 tags: [raster-ts-wrksp, raster]
 mainTag: raster-ts-wrksp
 description: "This lesson explores how to import and plot a multi-band raster in
-R. It also covers how to plot a three band color image plotRGB command in R"
-code1: 
+R. It also covers how to plot a three-band color image using the `plotRGB()`
```

Comment:
How about we create an issue. I don't normally include the () when i refer to a function. I believe it's that way in ALL of our NDS lessons. I think Data carpentry and SWC do that too but it's worth looking at their lessons. So changes will need to go one way or the other. I use use fewer characters so `functionName` rather than `functionName()`

https://github.com/NEONInc/NEONDS-Template-Repository/issues/8